src/mds/OpenFileTable.cc

```diff
@@ -725,6 +725,21 @@ void OpenFileTable::_recover_finish(int r)
   waiting_for_load.clear();
 }
 
+void OpenFileTable::_read_omap_values(const std::string& key, unsigned idx,
```

Comment:
```suggestion
void OpenFileTable::_read_omap_values(std::string_view key, unsigned idx,
```

Reply:
Fixed it.

It intended to be comatiable with the `op.omap_get_vals(const std::string &key, ...)`.

Reply:
I see. The change I suggested will end up creating unnecessary string temporaries. Let's undo this.

Reply:
Done.